docs/petalibrary/allocation_types.md

```diff
@@ -23,20 +42,50 @@ The archive feature list is similar to active, with the major differences being:
 
-Historically RC offered this service when the active and archive tiers were using different underlying storage systems, which made it difficult to do daily replication. The service was not available to customers for several years, but due to recent technical changes, we are able to not only bring back the service, but are also able to replicate data between the two tiers every fifteen minutes.
+The benefits of purchasing an active+archive allocation over purchasing your own active and archive allocations:
+* _No object limit_: a standalone archive allocation is subject to the 10,000 object (file/directory) limit. Since active allocations do not have an object limit, the archive component of an active+archive allocation also does not have an object limit.
+* _Automatic replication_: We replicate the data between the two tiers for you, every fifteen minutes.
 
 The means of replication between the active and archive tiers uses ZFS snapshots. The replication process copies any snapshots that exist on the active tier but do not exist on the archive tier, and likewise removes any snapshots that exist on the archive tier but are not present on the active tier. Therefore, if you ask us to prune ZFS snapshots aggressively on your active+archive allocation, this reduces the snapshot count on both the active and the archive tier. The active+archive tier is not an incremental backup, it is a time-delayed mirror of the ZFS snapshots of your active allocation. Snapshots are the best way to preserve your historical data in incremental steps, and replication ensures that your snapshots exist in two physical buildings. Physical isolation is important for events like fire and water damage that could damage one copy of your data beyond hope of recovery.
 
-## Archive + DR (Disaster Recovery)
+````
+
+````{tab-item} Archive+DR
+:sync: tabset-ucb-pl-tier-types-archive-dr
+
+The PetaLibrary Archive tier is configured to maximize data integrity over performance. Its triple parity data protection makes it a robust system with respect to disk failures. Although the Archive allocation tier is robust, all current allocations within this tier are hosted in a single location. In the event of a disaster at this location, we cannot guarantee the safety of this data. To add an extra layer of protection, we offer the Archive + DR (Disaster Recovery) tier. The Archive + DR tier offers the same benefits as the Archive tier in addition to a monthly backup of the data to an offsite location. The backup process and any needed restoration efforts will be managed entirely by CURC staff.
+
+````
+
+`````
+
+
+## Data Integrity Terminology
+
+::::{dropdown} Show data integrity terminology definitions
```

Comment:
Sorry if I wasn't clear enough in my previous comment. My suggestion was to either keep the `Data Integrity Terminology` section in its entirety or make the content a dropdown and place this dropdown underneath the table (that way it is close to the table and you do not have to go to another section). 

If you want to keep this a section, then I would suggest removing the dropdown and just keep it as a full section.